{tools}[GCCcore/11.3.0] pstoedit v3.78

sassy-crick opened the pull request:
(created using `eb --new-pr`)

Requires:
- [x] PR #15961
- [x] PR #15944
- [x] PR #16188 
- [x] PR #16196

jfgrimm commented:
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
himem01.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Platinum 8160 CPU @ 2.10GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/38cb81ad658174c314c75d976986365c for a full test report.

jfgrimm commented:
@boegelbot please test @ generoso

jfgrimm commented:
@boegelbot please test @ jsc-zen2

jfgrimm commented:
@sassy-crick looks like you're missing a dependency on ghostscript:
`configure: error: ** Cannot find ghostscript.  If it is already installed, Check PATH.`

sassy-crick commented:
> @sassy-crick looks like you're missing a dependency on ghostscript: `configure: error: ** Cannot find ghostscript. If it is already installed, Check PATH.`

There was a bit more to be done, so thanks for looking into that. ImageMagick was searched but not found, so that is what the patch is for. I have added PostScript as requested but I could not find anything in the configure step which is requiring PostScript. There are a few OS dependencies which I have added. I could not fine the PRM equivalent for `libplot-dev` so I left that out. Please have an eagle-eye look at this again and thanks for your help!

akesandgren commented:
@boegelbot please test @ jsc-zen2

jfgrimm commented on easybuild/easyconfigs/p/pstoedit/pstoedit-3.78-GCC-11.3.0.eb:
```diff
@@ -0,0 +1,50 @@
+    ('Ghostscript', '9.56.1'),
+]
+
+osdependencies = [('libemf-dev', 'libEMF-devel', 'libplot-dev')]
```
I don't see a reason not to create easyconfigs for these:
https://sourceforge.net/projects/libemf/
https://ftp.gnu.org/gnu/plotutils/

akesandgren replied:
I second that.

sassy-crick replied:
And I done that. Now you just have a few more PRs open. :-)
For the plotutils: that is quite old code and not really maintained. I am wondering if that is a good idea to have it in there, or if there is a replacement for that as well.

sassy-crick commented:
@boegelbot please test @ generoso

sassy-crick commented:
@boegelbot please test @ generoso

jfgrimm commented:
@boegelbot: please test @ jsc-zen3

jfgrimm commented:
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 5 out of 5 (1 easyconfigs in total)
node115.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/2dcfbdacf2e2dccf2de7f4abc7ec42f9 for a full test report.

jfgrimm commented:
Going in, thanks @sassy-crick!
